erpnext/www/all-products/index.py

```diff
@@ -19,6 +19,8 @@ def get_context(context):
 	for item in context.products:
 		item["active_batch"] = get_active_batch(item.name)
 		item["info"] = get_product_info_for_website(item.name)
+		itemgroup = frappe.db.get_value('Item', item.name, 'item_group')
+		item["group"] = ''.join(itemgroup.split()).lower()
```

Comment:
Isn't this already a string? Why do we need to split and re-join?

Reply:
I am just making it lowercase and removing spaces, just to avoid any kind of spelling differences while comparing

Reply:
Why are we removing the spaces though?

If you do wanna do it though, you can just use `frappe.scrub`, which lowercases strings and converts spaces into underscores:

```suggestion
		item_group = frappe.db.get_value('Item', item.name, 'item_group')
		item["group"] = frappe.scrub(item_group).replace("_", " ")
```